SparkPost/matchbox
# FE-941 Add root font size token

### What Changed
- Added some very basic tests to verify the token build works as expected
- Publishes design tokens
- Uses the new root font size token in matchbox global styles

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Inspect the page content, verify `html` has `16px` set as a font size

### PR Checklist
